src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -11,6 +11,7 @@
 using Azure.Functions.Cli.Common;
 using Azure.Functions.Cli.Diagnostics;
 using Azure.Functions.Cli.Extensions;
+// using Azure.Functions.Cli.Extensions;
```

Comment:
nit: remove if no longer used.

Reply:
Oh, my bad, that wasn't supposed to be there.